.github/workflows/delivery-archlinux-git.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
This pipeline just picks up the code from main, and shouldn't need to be versioned at all. This just serves as a trigger to start the pipeline if something goes wrong.